packages/react/src/components/pickers/BasePicker.types.ts

```diff
@@ -44,6 +44,16 @@ export interface IBasePickerProps<T> extends IReactProps<any> {
    */
   componentRef?: IRefObject<IBasePicker<T>>;
 
+  /**
+   * Descriptive label for the field
+   */
+  label?: string;
+
+  /**
+   * Aria Label for the field for screen reader users.
+   */
+  ariaLabel?: string;
```

Comment:
I get why you added this because Textfield does have it, but we now see it as an antipattern to add custom aliases for ARIA props. I think it'd be good to leave it off new additions to avoid adding more instances of this sort of attribute. Since we already support the native `aria-label`, I don't think BasePicker needs to also add `ariaLabel`.

Let me know if I'm missing a use case for this though!